dbusmock/templates/bluez5.py

```diff
@@ -186,7 +239,7 @@ def AddDevice(self, adapter_device_name, device_address, alias):
 
 @dbus.service.method(BLUEZ_MOCK_IFACE,
                      in_signature='ssi', out_signature='')
-def PairDevice(_self, adapter_device_name, device_address, class_=5898764):
+def PairDevice(self, adapter_device_name, device_address, class_=5898764):
```

Comment:
Why rename `_self` to `self`? If it's needed, it should probably happen in a separate commit.